src/Button/style.css

```diff
@@ -192,3 +192,7 @@
     padding-top: 4px;
     padding-bottom: 4px;
 }
+
+summary:focus .ola_button {
+  box-shadow: 0 0 0 4px var(--focus);
+}
```

Comment:
When button is not a `<a>` nor a `<button>` we should force the cursor to be `default`. Something like this:

```css
.ola_button:not(button):not(a) {
    cursor: default;
}

Reply:
In addition to this, I think we might have a `open` status for buttons inside a tooltip. That style could be the samen than `:hover` status, so we only have to add an extra selector to that style:

```css
details[open] summary .ola_button {
    /* Same styles than .ola_button:hover */
}
```

And the same for `ButtonIcon`.